SPECIFICATION.md

```diff
@@ -350,17 +350,17 @@ Lets assume we receive a feed id through an external channel (such as text messa
 
 ## Block prioritization
 
-When deciding which block to download from a remote peer a couple of things should be taken into suggestion.
+When deciding which block to download from a remote peer, a couple of things should be taken into account.
 
-To allow for features such as live booting of linux containers, real-time video and audio streaming, and any many more Hyperdrive allows for higher prioritization of specific ranges of blocks to download.
+To allow for features like live booting linux containers, real-time video and audio streaming, etc... Hyperdrive allows for higher prioritization of specific ranges of blocks to download first.
 
-If no blocks ranges are prioritized a strategy such as "Rarest first" or "Random first piece" should be used similar to the strategy used in BitTorrent. This allows blocks to be spread out evenly across peers without having single peers that are the only uploaders of a specific piece.
+If no blocks ranges are prioritized, a strategy such as "Rarest first" or "Random first piece" should be used similar to the strategy used in BitTorrent. This allows blocks to be spread out evenly across peers without having single peers that are the only uploaders of a specific piece.
 
-If a range is prioritized then blocks should be chosen from this range first. Ranges can be prioritized with different weights, CRITICAL, HIGH, NORMAL. For example if we are streaming video we might want to prioritize the first megabyte of the video as CRITICAL as the playback latency depends on this. We might also want to prioritize the next 10 mb with HIGH as we want to download that as a buffer. If a range is marked as CRITICAL it is also acceptable to make requests for the same block to multiple peers assuming a high bandwidth peer has no other blocks to request in the same range and has considerable more bandwidth than the peer, the block is currently being requested from. This is referred to as "hotswapping".
+If a range is prioritized, then blocks should be chosen from this range first. Ranges can be prioritized with different weights, CRITICAL, HIGH, NORMAL. For example if we are streaming video we might want to prioritize the first megabyte of the video as CRITICAL as the playback latency depends on this. We might also want to prioritize the next 10 mb with HIGH as we want to download that as a buffer. If a range is marked as CRITICAL it is also acceptable to make requests for the same block to multiple peers assuming a high bandwidth peer has no other blocks to request in the same range and has considerable more bandwidth than the peer, the block is currently being requested from. This is referred to as "hotswapping".
```

Comment:
If you are requesting a critical block from a peer and a newer peer comes along that has much higher bandwidth available but no free critical blocks to fetch then it's ok to do a request for the same block to this new better peer. 

Does that make sense?